utilities/cli/tests/test_cli.py

```
@@ -286,35 +286,18 @@ def test_lint_fix_command(self):
 
     @unittest.skipIf(not is_cookiecutter_available(), "cookiecutter not installed")
     @patch("utilities.cli.holohub.HoloHubCLI._install_template_deps")
-    @patch("cookiecutter.main.cookiecutter")
-    @patch("utilities.cli.holohub.HoloHubCLI.handle_create")
+    @patch("cookiecutter.main.cookiecutter", return_value="/tmp/dummy_project")
```

Ruff (0.15.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/dummy_project"

(S108)